cfgov/jinja2/v1/_includes/molecules/global-search.html

```diff
@@ -87,6 +87,14 @@
                 </ul>
             </div>
         </form>
-        <button class="m-global-search_tab-trigger" aria-hidden="true"></button>
+        <button class="m-global-search_tab-trigger u-visually-hidden"
+                aria-hidden="true">
+          .
```

Comment:
Change text to read "Collapse" instead of ".", and adjust the code comment for it.

Reply:
Fixed